M/Madagascar/build_tarballs.jl

```diff
@@ -3,37 +3,54 @@
 using BinaryBuilder, Pkg
 
 name = "Madagascar"
-version = v"1.0.0"
+version = v"1.0.1"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/ahay/src.git", "3a10302c63c69317df3f6f741347e7f77a2f3d58")
+    GitSource("https://github.com/ahay/src.git", "3a10302c63c69317df3f6f741347e7f77a2f3d58"),
+    DirectorySource("./bundled")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/src
+# Add host gcc for makefont
```

Comment:
We already have a compiler for the host, it's `${HOST_CC}`

Reply:
Right, it's without the underscore: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/c7830a667b5d28901eb6faa757db2c8a5ee043f4/src/Runner.jl#L1206

Reply:
Oh, that was even documented: https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables

Reply:
Ok fixed

Reply:
> Ok fixed

I still see the `apk add` command below 🙂